scrapy/mail.py

```diff
@@ -12,7 +12,9 @@
 from email.utils import formatdate
 from io import BytesIO
 
+from incremental import Version
```

Comment:
We usually do `from twisted.python.versions import Version` (even though it's the same class)